sfm/ui/templates/ui/collection_set_detail.html

```diff
@@ -91,22 +91,26 @@
                       Add Collection <span class="caret"></span>
                 </button>
-                <a target="_blank" href="http://sfm.readthedocs.io/en/latest/collections.html">
-                    <span class="glyphicon glyphicon-info-sign"  style="padding-top:7px;  margin-left: 10px"></span>
-                </a>
-                <ul class="dropdown-menu">
-                    {% for harvest_type, harvest_label in harvest_types %}
-                        <li><a data-toggle="tooltip" data-placement="right"
-                                                                            title="{% if 'twitter_user_time' in harvest_type %}Tweets from specific accounts{% endif %}
-                                                                                   {% if 'twitter_search' in harvest_type %}Recent tweets matching a query{% endif %}
-                                                                                   {% if 'twitter_filter' in harvest_type %}Tweets in real time matching filter criteria{% endif %}
-                                                                                   {% if 'twitter_sample' in harvest_type %}A subset of all tweets in real time{% endif %}
-                                                                                   {% if 'flickr_user' in harvest_type %}Posts and photos from specific accounts{% endif %}
-                                                                                   {% if 'weibo_timeline' in harvest_type %}Posts from a user and the user's friends{% endif %}
-                                                                                   {% if 'tumblr_blog_posts' in harvest_type %}Blog posts from specific blogs{% endif %}" 
-                               href="{% url "collection_create" collection_set.id harvest_type %}">Add {{ harvest_label }}</a></li>
-                    {% endfor %}          
-                </ul>
+
+                <a target="_blank" href="http://sfm.readthedocs.io/en/latest/collections.html" data-toggle="tooltip" title="Collections connect to a social media platform's APIs, or methods for retrieving data. Understanding what each collection type provides is important to ensure you collect what you need and are aware of any limitations. Click for more info"><span class="glyphicon glyphicon-info-sign" style="padding-top:7px; margin-left: 10px"></span></a>
+   
+                     <ul class="dropdown-menu">
+                          {% for harvest_type, harvest_label in harvest_types %}
+                          <li><a  href="{% url "collection_create" collection_set.id harvest_type %}" class="flex-column align-items">
+                               <h4>Add {{ harvest_label }}</h4>
+                               <p class="mb-1">
+                                            {% if 'twitter_user_time' in harvest_type %}Tweets from specific accounts{% endif %}
```

Comment:
This is the right order for the collection types, but I'm not seeing them appear in this order in the UI. Do you know why?